### ISSUE TITLE
AUXPOW activated?

### ISSUE THREAD

**Issue by p3yot3** [lodyagin/compcoin#2]
Has AUXPOW been activated for merge mining?

**Comment by lodyagin** [comment]
I believe merged-mining works. We used container compcoin/merged-mining/Dockerfile for testing. Please let me know if you find some issues with that. Do I answer your question?

**Comment by p3yot3** [comment]
I've been merge mining it for the last day but don't think it's working, diff has barely moved from 11 & I've not seen any blocks found or paid. Here are the daemon logs:

2017-06-16 17:21:09 ThreadRPCServer method=getauxblock
2017-06-16 17:21:09 getauxblock
2017-06-16 17:21:10 ThreadRPCServer method=getauxblock
2017-06-16 17:21:10 getauxblock
2017-06-16 17:21:11 ThreadRPCServer method=getauxblock
2017-06-16 17:21:11 getauxblock
2017-06-16 17:21:12 ThreadRPCServer method=getauxblock
2017-06-16 17:21:12 getauxblock
2017-06-16 17:21:13 ThreadRPCServer method=getauxblock
2017-06-16 17:21:13 getauxblock
2017-06-16 17:21:14 ThreadRPCServer method=getauxblock
2017-06-16 17:21:14 getauxblock
2017-06-16 17:21:15 ThreadRPCServer method=getauxblock
2017-06-16 17:21:15 getauxblock
2017-06-16 17:21:16 ThreadRPCServer method=getauxblock
2017-06-16 17:21:16 getauxblock
2017-06-16 17:21:17 ThreadRPCServer method=getauxblock
2017-06-16 17:21:17 getauxblock
2017-06-16 17:21:18 ThreadRPCServer method=getauxblock
2017-06-16 17:21:18 getauxblock
2017-06-16 17:21:19 ThreadRPCServer method=getauxblock
2017-06-16 17:21:19 getauxblock
2017-06-16 17:21:20 ThreadRPCServer method=getauxblock
2017-06-16 17:21:20 getauxblock

I'm also seeing lot's of block submittal failures in the p2pool logs:

2017-06-16 18:25:28.594031 > Merged block submittal result: False Expected: True
2017-06-16 18:25:38.299237 > Merged block submittal result: False Expected: True
2017-06-16 18:25:52.338422 > Merged block submittal result: False Expected: True
2017-06-16 18:25:58.217350 > Merged block submittal result: False Expected: True
2017-06-16 18:26:04.386733 > Merged block submittal result: False Expected: True
2017-06-16 18:26:51.015287 > Merged block submittal result: False Expected: True
2017-06-16 18:26:51.632289 > Merged block submittal result: False Expected: True
2017-06-16 18:28:28.472447 > Merged block submittal result: False Expected: True

Here's the daemon info:

 "version" : 1034204,
    "protocolversion" : 70003,
    "walletversion" : 0,
    "balance" : 0,
    "blocks" : 91719,
    "timeoffset" : 0,
    "connections" : 7,
    "proxy" : "",
    "difficulty" : 15.78534068,
    "testnet" : false,
    "keypoololdest" : 1497524720,
    "keypoolsize" : 101,
    "paytxfee" : 0,
    "mininput" : 1000,
    "errors" : ""

**Comment by p3yot3** [comment]
Any ideas?

BTW - which of these is you:

https://bitcointalk.org/index.php?topic=981831.new#new

https://bitcointalk.org/index.php?topic=1965246.0

**Comment by lodyagin** [comment]
OK, thank you for the details. I will investigate the issue but it seems a few people are trying merged-mining just now, so, I should say "sorry" and ask you to wait a bit. If you really urgently need it, please contact the compcoin company - it is the owner of the currency and I am only a developer.

Neither the first nor the second. I have no account there (should I ? :).

**Comment by lodyagin** [comment]
> f you really urgently need it,  please contact the compcoin company

Of course I will be happy to forward your request. The point is I don't know their priorities, so it is better to talk with them if you have some big interest. But, of course, from my side, I will investigate the issue if I will not have some urgent requests from them.

**Comment by p3yot3** [comment]
I'm sorry, I don't quite understand.

Which compcoin company should I contact if you are neither of the ones I asked you about?   Is there another one?   Can you provide me a link?

**Comment by lodyagin** [comment]
The compcoin company has a website http://compcoin.com. If you have some business interests and for that want to fix things urgently you should contact them. Just because requests from them as from a client to me, as a currency developer, always are of the highest priority. 
If you can wait - no problem. I will investigate the issue when I will have time. Also, I can put the issue in the list of my proposition to them, but to be able to do so I need a few more details about you and your MM environment (you can treat me as a technical guy behind compcoin currency, who has his own opinion what to do and how).

**Comment by p3yot3** [comment]
OK. Wait a minute.

You're the dev of a coin that has been released even though it's not working & you're not sure how to fix it?

Why do you want my details? That's not a requirement here nor should it be.

The enviroment is as I've posted already: p2pool on linux using the compcoin daemon. I've posted all logs already - what more environment details do you require?

The website link you provided takes me to some dodgy ICO site - there's zero info there about mining or any specs about the coin.

**Comment by lodyagin** [comment]
"The website link you provided takes me to some dodgy ICO site - there's zero info there about mining or any specs about the coin."
- So, that should be fixed first. I will work on that. I am really not a person who should be contacted first.

"You're the dev of a coin that has been released even though it's not working & you're not sure how to fix it?"
- What's wrong with that? We didn't test AUXPOW with p2pool as far as I know. I believe your issue could be fixed. If you need it urgently please contact the company, I don't put priorities on tasks by my own. 

"Why do you want my details? That's not a requirement here nor should it be."
Just to forward your request if you don't want to contact the people who manage the things.

**Comment by p3yot3** [comment]
I don't think you understand. Github is designed so that developers (like yourself?) & contributors (me) can colaborate & report/solve any issues that become apparent. A develper telling github users to contact some fishy ICO site to report issues is not only suspect, but makes no sense whatsoever - github is used for this exact thing. It's the whole point of github.

AUXPOW is a standardised protocol which is compatible across all mining pools - if it works on one pool, it works on all pools, on all platforms. Surely as the developer of this coin you know this?

I will not contact any fishy ICO site for a coin who's wallet does not work or have my details passed to them by anyone else, thank you.

I suggest you fix your coin before fixing your fishy ICO website - anyone who is foolish enough to buy into your ICO will be very upset when they realise nothing works & the dev (?) doesn't know why.

I will delete this garbage from my systems immediately.

**Comment by lodyagin** [comment]
Sorry to hear that. But it's how this currency was designed, it addresses a specific segment of a market. I agree, it has nothing with usual open-source practices on github, so I will remove it from here. The owner should be contacted for all urgent questions including access to sources. That is not my decision, sorry.
